torch/_higher_order_ops/invoke_subgraph.py

```diff
@@ -580,6 +581,26 @@ def backward(
         for tangent in filtered_grad_outs:
             metadata = extract_tensor_metadata(tangent)
             metadata._flatten_into(tangent_metadata, fake_mode, state)
+
+        # Add aliasing information to tangent_metadata
+        # Two tangents are aliased if they are the same tensor object (using id())
+        # We create a tuple of tuples where each inner tuple contains indices of aliased tensors
+        # e.g. ((0, 1),) would mean there is one aliasing group, and the first and second tangents are aliased
+        # e.g. () would mean there is no aliasing between tangents
+        tensor_to_indices: dict[int, list[int]] = defaultdict(list)
+        for i, tangent in enumerate(filtered_grad_outs):
+            if isinstance(tangent, torch.Tensor):
```

Comment:
note: the tensors here are FakeTensors, so I just check id() not their storage

Reply:
Why not check storage? It might be possible to get two gradients, one of which is `grad` and the other `grad[0]`.